docs/reference/indices/close.asciidoc

```diff
@@ -34,7 +34,7 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index]
 +
 To close all indices, use `_all` or `*`.
 To disallow the closing of indices with `_all` or wildcard expressions,
-change the `action.destructive_requires_name` cluster setting to `true`.
+the `action.destructive_requires_name` cluster setting must be `true`.
```

Comment:
As above, I think it's worth rewording this paragraph so that the default behavior is explained before the configuration option.

The same recommendation applies to the other docs files.

Reply:
Done.